_episodes/07-tasks-and-automation.md

```diff
@@ -1,19 +1,114 @@
-teaching: 0
-exercises: 0
+teaching: 20
+exercises: 2
 questions:
 - "What are tasks?"
-
+- "How do you manage and create new tasks?""
+- "How do you run tasks?""
 
 objectives:
-- "Explain the objective"
-
+- "Explain how to create, manage, edit and run tasks"
+- "Successfully create and run a new task"
+- "Successfully export a task"
 
 keypoints:
-- "First key point."
+- "MarcEdit tasks are a way to automate a series of functions."
+- "Tasks can consiste of any type of function offered in the MarcEditor tools"
 ---
-{% include links.md %}
-
 
 # Tasks and Automation
+Very often we deal with a set of MARC records where the data needs the same or very similar approach. A good example are electronic resources where the resource link in the MARC field 856 will need the proxy stem for that institution. Another example are adding RDA content, carrier, and media terms and codes to your records. In short, MARC records need to fit both national standards and local best practices and hence it is necessary to manipulate MARC data to fit these standards and practices. In the previous lessons we saw this at the scale of one change at a time. In this lesson, we move to seeing how to string together these changes.
+
+Indeed, a MarcEdit task is a series of changes to MARC data that occur in the sequence they are entered in the task. One could think of tasks as a mini script of sorts that string together different MarcEditor tools that manipulate MARC data in some way. All tasks are created, managed, and run in the MarcEditor. In this way, tasks are another way to manipulate data in the MarcEditor and are an extension of the other tools offered in the MarcEditor.
+
+## What are tasks?
+A task is a series of MarcEdit functions to manipulate data. In the previous lessons, we've run one function at a time. Tasks string those functions together to run at once. Tasks run each function one after another. As such, it is important to think about which tasks follow other tasks. For instance, if you want to add a proxy and the MARC data already has a proxy that you need to delete, your task has to account for that and remove the old proxy before adding a new one.
+
+Tasks are managed through the Task Manager in the MarcEditor. To access the Task Manager, go to Tools and then Manage Tasks in the top level menu. This will open the new tasks window.
+
+In this window, you will see any tasks that you have created previously. You will see the name that you gave those tasks, the path or where those files are saved on your local computer, a shortcut, and a description. At the bottom of this window you will also have two options "Task Actions" and "Manage Existing Tasks".
+
+Task Actions include:
+- New Task: This will create a new task.
+- Clone: This will duplicate an existing task.
+- Delete: This will remove a task.
+- Edit: This will allow you to edit an existing task.
+- Rename: This is how to give a different name to an existing task.
+
+Manage Existing Tasks include:
```

Comment:
Not certain the Manage Existing Task list needs to be delineated like this if we're not covering groups key strokes etc. Maybe just focus on what we're covering?